decidim-elections/spec/system/vote_online_spec.rb

```diff
@@ -39,6 +39,13 @@
 
       uses_the_voting_booth
     end
+
+    it "shows a link to view more information about the election" do
```

Comment:
Can you add the regression spec for not showing the More information link when there isn't any description 🙏🏽?

Reply:
Done!